docs/fundamentals/diagnostics/sample/src/DiagDemo/project.lock.json

```diff
@@ -0,0 +1,1146 @@
+{
```

Comment:
@danroth27 do we want to check in lock files for samples? We don't do that elsewhere.